.eslintrc.js

```diff
@@ -10,17 +10,11 @@ module.exports = {
       jsx: true
     }
   },
-  plugins: [
-    'react',
-    'prettier',
-    'jsdoc'
-  ],
-  extends: [
-    'plugin:react/recommended',
-    'airbnb',
-    'prettier',
-    'plugin:jsdoc/recommended'
-  ],
+  globals: {
+    expect: 'readonly'
+  },
```

Comment:
I'm not sure what this is trying to fix?

Reply:
It looks like it's related to the `expect` function from `vitest` and ignores those errors if added. Not that's not needed in this case, we can revert this change back.